Quadriga/src/main/java/edu/asu/spring/quadriga/web/workbench/RetrieveProjectController.java

```diff
@@ -168,39 +169,29 @@ public String getProjectDetails(@PathVariable("projectid") String projectid, Pri
         model.addAttribute("deactivatedWSSize", deactivatedWSSize);
         model.addAttribute("archivedWSSize", archivedWSSize);
 
+        Map<String, Boolean> collaboratorRoles = projectSecurity.getCollaboratorRoles(userName, projectid);
+
         if (projectSecurity.isProjectOwner(userName, projectid)) {
             model.addAttribute("owner", 1);
         } else {
             model.addAttribute("owner", 0);
         }
-        if (projectSecurity.isEditor(userName, projectid)) {
+        if (collaboratorRoles.get(RoleNames.ROLE_PROJ_COLLABORATOR_EDITOR)) {
             model.addAttribute("editoraccess", 1);
         } else {
             model.addAttribute("editoraccess", 0);
```

Comment:
since you are using true/false for the other roles, why not changing that here as well?